### ISSUE TITLE
Hide integration from CI matrix

### ISSUE THREAD

**Issue by seberg** [gerrymanoim/pr-prefix-labeler#16]
Not sure if you can hide github actions from the CI matrix, but it seems unnecessary for it to show up there, so I would prefer if it were just hidden. It already is visible as the bot adding the actual label, which seems good enough.

**Comment by gerrymanoim** [comment]
I agree with you - unfortunately, as best I can tell, there's no way for the action to hide itself from the CI matrix.
